rosbag_snapshot/src/snapshotter.cpp

```diff
@@ -562,6 +634,14 @@ void Snapshotter::pollTopics(ros::TimerEvent const& e, rosbag_snapshot::Snapshot
     for (ros::master::TopicInfo const& t : topics)
     {
       std::string topic = t.name;
+      SnapshotterTopicOptions topic_options;
+      if (!options->all_topics_)
```

Comment:
Can you add a comment here to the effect of `if we are not recording all topics, we have to check to see if this topic matches any of the patterns`